lib/auth/auth.go

```diff
@@ -1659,10 +1661,16 @@ type newMFADeviceFields struct {
 	// Identity with an in-memory SessionData storage.
 	// Defaults to the Server's IdentityService.
 	webIdentityOverride wanlib.RegistrationIdentity
+	// deviceResp is the register response from the new device.
+	deviceResp *proto.MFARegisterResponse
+	// passwordless explicitly requests to register a passwordless device.
+	// This flag will be used to verify that the register response is
+	// passwordless capable.
+	passwordless bool
```

Comment:
Do you think it's worth passing the DeviceUsage instead, only turning it into a boolean in the last method?